src/stylesheets/components/_hero.scss

```diff
@@ -2,36 +2,36 @@
 // ==========================
 
 .usa-hero {
-  @include u-padding-y($site-margins);
+  @include u-padding-y(units(4));
   background-image: url('#{$theme-image-path}/hero.png');
   background-position: center;
   background-size: cover;
 
   .usa-button {
-    color: $color-white;
+    color: color('white');
   }
 }
 
 .usa-section-dark {
   .usa-hero-link {
-    color: $color-primary-alt;
+    color: color('accent-cool');
   }
 }
 
 .usa-hero-callout {
-  background-color: $color-primary-darkest;
-  padding: 3rem;
+  background-color: color('base-darker');
+  padding: units(4);
 
-  @include media($small-screen) {
+  @include media(units('mobile-lg')) {
```

Comment:
Technically `small-screen` covers 481px, and our new `mobile-lg` starts at 480px, so not 100% sure if this is the right sub

Reply:
It's not 100% exact, but I wonder if it will matter? Also, I recommend using the new mixin for media queries: `@include at-media('mobile-lg')`

Reply:
Looks like this still needs to be done.

Reply:
nah, I changed it to use the `at-media` rule

Reply:
units is still there, it should look like how @thisisdano wrote it above.